zerver/models.py

```diff
@@ -999,7 +999,7 @@ def content_has_image(content):
 
     @staticmethod
     def content_has_link(content):
-        return 'http://' in content or 'https://' in content or '/user_uploads' in content
+        return 'http://' in content or 'https://' in content or '/user_uploads' in content or 'file:///' in content
```

Comment:
hmm probably this `or` case should be conditional on `settings.ENABLE_FILE_LINKS` as well, right?